pyupgrade.py

```diff
@@ -23,10 +40,29 @@
 from tokenize_rt import UNIMPORTANT_WS
 
 
+Parsed = Tuple[Text, Optional[Text], Optional[Text], Optional[Text]]
+Fmt = Tuple[
+    Optional[Text],
+    Optional[Text],
+    Optional[Text],
+    Optional[Text],
+    Optional[Text],
```

Comment:
I think this one is non-optional

Reply:
It's a hack to support https://github.com/asottile/pyupgrade/blob/c0642054382847c707ec65ec070ffadd89c3dabd/pyupgrade.py#L736

Reply:
hmmm! can that line be changed to `conversion = ''` and still work?

Reply:
I guess it passed tests? I haven't really tried to understand the code as well I could have 🙈